tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/HotChocolate/HotChocolateCommon.cs

```diff
@@ -142,17 +146,93 @@ private static string ConstructErrorMessage(List<IError> executionErrors)
                     if (message != null)
                     {
                         builder.AppendLine($"{tab + tab}\"message\": \"{message.Replace("\r", "\\r").Replace("\n", "\\n")}\",");
+                        eventAttributes.Add(new KeyValuePair<string, object>("message", message));
+                    }
+
+                    var locations = executionError.Locations;
+                    if (locations != null)
+                    {
+                        ConstructErrorLocationsMessage(builder, tab, locations);
+
+                        var joinedLocations = new List<string>();
+                        foreach (var location in (System.Collections.IEnumerable)locations)
+                        {
+                            if (location.TryDuckCast<ErrorLocationStruct>(out var locationProxy))
+                            {
+                                joinedLocations.Add($"{locationProxy.Line}:{locationProxy.Column}");
+                            }
+                        }
+
+                        eventAttributes.Add(new KeyValuePair<string, object>("locations", joinedLocations.ToArray()));
                     }
 
-                    ConstructErrorLocationsMessage(builder, tab, executionError.Locations);
                     builder.AppendLine($"{tab}}},");
+
+                    var path = executionError.Path;
+                    if (path.Name != null)
+                    {
+                        var pathAttribute = new List<string>();
+                        pathAttribute.Add(path.Name);
+                        eventAttributes.Add(new KeyValuePair<string, object>("path", pathAttribute.ToArray()));
```

Comment:
Could swap this to something like so to get rid of the `new List<string>` which appears to be not needed

```suggestion
                        eventAttributes.Add(new KeyValuePair<string, object>("path", new[] { path.Name }));
```